scripts/run.sh

```diff
@@ -65,9 +65,24 @@ echo "Test pod exited with code:${podExitCode}"
 #Collect artifacts
 retrive-artifacts
 retrieveArtifactsExitCode=$?
+echo "Retrieving artifacts exited with code ${retrieveArtifactsExitCode}"
 
 
 #Clean up
 delete_lease
-exitCode=$(${podExitCode}+${retrieveArtifactsExitCode})
-exit $exitCode
+deleteLeaseExitCode=$?
+echo "Deleting lease exited with code ${deleteLeaseExitCode}"
+
+if [ "$retrieveArtifactsExitCode" -ne "0" ]; then
+  exit ${retrieveArtifactsExitCode}
+fi
+
+if [ "$retrieveArtifactsExitCode" -ne "0" ]; then
```

Comment:
thx @vdice I replaced the duplicated code with a pod exit code check in https://github.com/deis/e2e-runner/pull/17/commits/e9c17e4ab50e698657baf564ccf918ff367afa8e